template-for-repository/proofs/Makefile.common

```diff
@@ -287,7 +287,11 @@ DEFINES += -DCBMC=1
 DEFINES += -DCBMC_OBJECT_BITS=$(CBMC_OBJECT_BITS)
 DEFINES += -DCBMC_MAX_OBJECT_SIZE="(SIZE_MAX>>(CBMC_OBJECT_BITS+1))"
 
-CBMC_UNWINDSET := $(patsubst %,--unwindset %, $(UNWINDSET))
+# CI currently assumes cbmc invocation has at most one --unwindset
+# CBMC_UNWINDSET := $(patsubst %,--unwindset %, $(UNWINDSET))
+ifneq ($(UNWINDSET),"")
+  CBMC_UNWINDSET = --unwindset $(subst $(SPACE),$(COMMA),$(strip $(UNWINDSET)))
```

Comment:
The previous version used `:=` rather than `=`. I'd prefer `:=` unless there's a good reason not to, because it's faster.